File: Programming-Basics-With-JavaScript/Homeworks-And-Labs/06.Nested-Loop/03-additional/05.challengeTheWedding.js
function challengeTheWedding(input) {

    let mans = Number(input[0]);
    let womans = Number(input[1]);
    let tables = Number(input[2]);
    let printLine = '';
    let table = 0;

    for (let i = 1; i <= mans; i++) {
        for (let j = 1; j <= womans; j++) {
            table++;
            if (table <= tables) {
                printLine += `(${i} <-> ${j}) `;
            }
        }
    }

    console.log(printLine);
}
challengeTheWedding(['5', '8', '40']);

File: Python_Basics/Homeworks-And-Labs/06.nested_loops/03.additional/05.challenge_the_wedding.py
mans = int(input())
womans = int(input())
tables = int(input())

print_line = ''
table = 0

for i in range(1, mans + 1):
    for j in range(1, womans + 1):
        table += 1
        if table <= tables:
            print_line += f'({i} <-> {j}) '
            
print(print_line)
